### ISSUE TITLE
About ST-GCN

### ISSUE THREAD

**Issue by 15762260991** [open-mmlab/mmaction2#2281]
I think this is an excellent job, but I have a question: the learning rate set by the authors in the paper is different from the one in the code, which one should I use?

**Comment by cir7** [comment]
@15762260991 
In my opinion, the best choice is to use the same hyperparameters as the paper, and as an experience, scaling the learning rate according to the actual batch size and original batch size is ok for most experiments.
